mmengine/fileio/backends/oss_backend.py

```diff
@@ -0,0 +1,455 @@
+            >>> backend.get_text(filepath)
+            'hello world'
+        """
+        return str(self.get(filepath), encoding=encoding)
+
+    def put(self, obj: bytes, filepath: Union[str, Path]) -> None:
+        """Write bytes to a given ``filepath``.
+
+        Args:
+            obj (bytes): Data to be saved.
+            filepath (str or Path): Path to write data.
+
+        Examples:
+            >>> backend = OSSBackend()
+            >>> filepath = 'oss://endpoint/bucket/file'
+            >>> backend.put(b'hello world', filepath)
+        """
+        endpoint, bucket_name, obj_name = self._parse_path(filepath)
+        bucket = self._bucket_instance(endpoint, bucket_name)
+        bucket.put_object(obj_name, obj)
+
+    def put_text(
+        self,
+        obj: str,
+        filepath: Union[str, Path],
+        encoding: str = 'utf-8',
+    ) -> None:
+        """Write text to a given ``filepath``.
+
+        Args:
+            obj (str): Data to be written.
+            filepath (str or Path): Path to write data.
+            encoding (str): The encoding format used to encode the ``obj``.
+                Defaults to 'utf-8'.
+
+        Examples:
+            >>> backend = OSSBackend()
+            >>> filepath = 'oss://endpoint/bucket/file'
+            >>> backend.put_text('hello world', filepath)
+        """
+        self.put(bytes(obj, encoding=encoding), filepath)
+
+    def exists(self, filepath: Union[str, Path]) -> bool:
+        """Check whether a file path exists.
+
+        Args:
+            filepath (str or Path): Path to be checked whether exists.
+
+        Returns:
+            bool: Return ``True`` if ``filepath`` exists, ``False`` otherwise.
+
+        Examples:
+            >>> backend = OSSBackend()
+            >>> filepath = 'oss://endpoint/bucket/file'
+            >>> backend.exists(filepath)
+            True
+        """
+        endpoint, bucket_name, obj_name = self._parse_path(filepath)
+        bucket = self._bucket_instance(endpoint, bucket_name)
+        exist = bucket.object_exists(obj_name)
+        if exist:
+            return True
+        else:
+            return False
+
+    def rmtree(self, dir_path: Union[str, Path]) -> None:
+        """Recursively delete a directory tree.
+        Attention: if dir_path endwith '/', this operator represents
+        delete all files and directory under src. otherwise, delete
+        all files and directory include src folder.
+
+        Args:
+            dir_path (str or Path): A directory to be removed.
+
+        Examples:
+            >>> backend = OSSBackend()
+            >>> dir_path = 'oss://endpoint/bucket/src'
+            >>> backend.rmtree(dir_path)
+        """
+        endpoint, bucket_name, obj_name = self._parse_path(dir_path)
+        bucket = self._bucket_instance(endpoint, bucket_name)
+        for obj in self.oss2.ObjectIterator(bucket, prefix=obj_name):
+            bucket.delete_object(obj.key)
```

Comment:
```suggestion
    def rmtree(self, dir_path: Union[str, Path]) -> None:
        """Recursively delete a directory tree.
        
        Note:
            If dir_path ends with '/', this operator will
            delete all files and directories under dir_path. otherwise, delete
            all files and directories including dir_path ifself.

        Args:
            dir_path (str or Path): A directory to be removed.

        Examples:
            >>> backend = OSSBackend()
            >>> dir_path = 'oss://endpoint/bucket/src'
            >>> backend.rmtree(dir_path)
        """
        endpoint, bucket_name, obj_name = self._parse_path(dir_path)
        bucket = self._bucket_instance(endpoint, bucket_name)
        for obj in self.oss2.ObjectIterator(bucket, prefix=obj_name):
            bucket.delete_object(obj.key)
```